spec/factories/host_switch.rb

```diff
@@ -0,0 +1,3 @@
+FactoryGirl.define do
+  factory :host_switch
```

Comment:
Why is this required?

Reply:
https://github.com/ManageIQ/manageiq-ui-classic/pull/3933/files#diff-c656f77797c57a2c5dc0688b27ef5565R16

Reply:
I added a hack so that we don't need to define all of these empty factories.

https://github.com/ManageIQ/manageiq/blob/master/spec/support/missing_factory_helper.rb#L1